data/models.py

```diff
@@ -120,6 +120,14 @@ def get_categories(self):
             self.category_list_cache = self._construct_categories()
         return self.category_list_cache
 
+    def get_name_cs(self):
+        """Get the case sensitive version of the variable name, if available"""
+        try:
+            name_cs = self.get_source["name_cs"]
+        except:
```

Comment:
Please make this except clause explicit: Which exception are you catching here? AttributeError, KeyError?

Please use
```python
except AttributeError:
    name_cs = self.name
```

Reply:
Hmm, I think, I can simply use the get() method. I'll push an update...